docs/deploy/includes/prerequisites.md

```diff
@@ -1,10 +1,13 @@
 ## Prerequisites
 
 {: .no_toc }
-A production-suitable lakeFS installation will require three DNS records **pointing at your lakeFS server**.
+
+Users that require S3 access using [virtual host addressing](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) will require three DNS records **pointing at your lakeFS server**.
```

Comment:
Perhaps link to the new doc page and not AWS?

Or maybe even remove this prerequisites section and leave just a link to the new doc page